.travis.yml

```diff
@@ -57,5 +57,6 @@ deploy:
   skip_cleanup: true
   on:
     tags: true
+    node_js: 10.7
```

Comment:
I think it's supposed to be just `node: 10.7` according to the travis docs https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on